app/controllers/jobs_controller.rb

```diff
@@ -51,15 +52,14 @@ def enabled
   end
 
   def destroy
```

Comment:
is this action called `destroy` elsewhere? we aren't marking a record as deleted, not sure if DELETE is the right http verb.